pkg/operator/ceph/test/containers.go

```diff
@@ -184,7 +184,7 @@ func isCephCommand(command []string) bool {
 func argEnvReferences(c *v1.Container) []string {
 	argRefSet := map[string]bool{}
 	for _, a := range c.Args {
-		argRefRegex, e := regexp.Compile("\\$\\(([a-zA-Z][a-zA-Z0-9_]*)\\)")
+		argRefRegex, e := regexp.Compile("\\$\\(([a-zA-Z][a-zA-Z0-9_]*)\\)") // nolint:gosimple
```

Comment:
here we getting an error 
```pkg/operator/ceph/test/containers.go:187:21: S1007: should use raw string (`...`) with regexp.Compile to avoid having to escape twice (gosimple)
		argRefRegex, e := regexp.Compile("\\$\\(([a-zA-Z][a-zA-Z0-9_]*)\\)")```

Solution to above error is ```argRefRegex, e := regexp.Compile(`\\$\\(([a-zA-Z][a-zA-Z0-9_]*)\\)`)``` but this leads to build error while `make build`.

Reply:
Sounds good. So the way to suppress individual errors is with `nolint:<ruleName>`?

Reply:
Yes, and changing it to `regexp.Compile(`\$\(([a-zA-Z][a-zA-Z0-9_]*)\)`)` is  working, I'll change this.